Server/Game/unithelper.py

```diff
@@ -177,5 +141,181 @@ def setTeam(leader_ability, perks, units, username, version):
 
 	return err_msg
 
-def validateMove(unit, game, newX, newY):
+def calculateAttack(game, unit_dict, target):
+
+	# Create the response object
+	response = {}
+	response["Unit"] = unit_dict
+	response["Unit"]["ID"] = response["Unit"]["Unit"].id
+	response["Unit"].pop("Unit", None)
+	response["Target"] = {"ID":target.id}
+
+	return response
+
+def calculateHeal(game, unit_dict, target):
+
+	# Create the response object
+	response = {}
+	response["Unit"] = unit_dict
+	response["Unit"]["ID"] = response["Unit"]["Unit"].id
+	response["Unit"].pop("Unit", None)
+	response["Target"] = {"ID":target.id}
+
+	return response
+
+def updateValidAction(game, unit_dict, target_dict=None):
+	"""
+	Updates the game with the provided action
+
```

Comment:
This method gets called at the end of the path for each action.  It will be updated to include HP for actions that have targets, but for ones without a target, HP is not needed.